components/DropdownLight/DropdownLight.unit.test.jsx

```diff
@@ -19,31 +19,37 @@ describe('<DropdownLight />', () => {
   it('should return a value from item, using a object items', () => {
     render(<DropdownLight items={itemsObjectMock} />);
 
-    const dropdown = screen.getByRole('combobox');
-    userEvent.selectOptions(dropdown, 'Lemon');
+    const dropdown = screen.getByRole('button');
+    userEvent.click(dropdown, 'Select an option');
```

Comment:
Why are you passing a second argument to the click function? I don't see the need for that

Reply:
To have access to the list, it is necessary to redeem its value after clicking on the button